helm/alfresco-content-services/charts/alfresco-elasticsearch-connector/templates/reindexing-job.yaml

```diff
@@ -0,0 +1,53 @@
+{{- if .Values.reindexing.enabled }}
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: {{ include "alfresco-elasticsearch-connector.fullname" . }}-reindexing
+spec:
+  template:
+    metadata:
+      {{- with .Values.podAnnotations }}
+      annotations:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      labels:
+        {{- include "alfresco-elasticsearch-connector.selectorLabels" . | nindent 8 }}
+    spec:
+      # only set this secret if a private docker registry variable is defined
+      imagePullSecrets:
+        - name: quay-registry-secret
+      restartPolicy: Never
+      containers:
+        - name: {{ .Chart.Name }}-reindexing
+          image: "{{ .Values.image.repositoryPrefix }}-reindexing:{{ .Values.image.tag }}"
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          envFrom:
+            - configMapRef:
+                name: {{ template "alfresco-elasticsearch-connector.fullName" . }}-elasticsearch-connector-reindexing-configmap
+          env:
+            -
+              name: SPRING_DATABASE_PASSWORD
+              valueFrom:
+                secretKeyRef:
+                  name: {{ template "content-services.shortname" . }}-dbsecret
+                  key: DATABASE_PASSWORD
+            -
+              name: SPRING_DATABASE_USERNAME
+              valueFrom:
+                secretKeyRef:
+                  name: {{ template "content-services.shortname" . }}-dbsecret
+                  key: DATABASE_USERNAME
+          ports:
+              - name: http
+                containerPort: 8080
+                protocol: TCP
+      initContainers:
+        - name: wait-for-repository
+          image: curlimages/curl:latest
+          env:
+            - name: ALFRESCO_REPOSITORY_URL
+              value: http://{{ template "alfresco.shortname" . }}-cs-repository/alfresco/api/-default-/public/alfresco/versions/1/probes/-ready-
+          command: [ "/bin/sh","-c" ]
+          # Delay running the reindexing to give Alfresco Repository a chance to fully initialise
+          args: [ "while [ $(curl -sw '%{http_code}' $ALFRESCO_REPOSITORY_URL -o /dev/null) -ne 200 ]; do sleep 5; echo 'Waiting for the Alfresco Repository...'; done; echo 'Alfresco is ready, delay reindexing to give a chance to fully initialise.'; sleep 180; echo 'Reindexing started!'" ]
```

Comment:
The sleep can be reduced, it is required to wait for the index initialization performed by Alfresco Repository at startup.  I'm going to reduce it during the first commit after the pull request review.